File: Faza6/OnlineGamesShop/src/main/java/com/rich/onlinegamesshop/model/reports/WeeklySalesReport.java
package com.rich.onlinegamesshop.model.reports;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class WeeklySalesReport {
    private String username;
    private  BigDecimal weekNumber;
    private Long totalOrders;
    private BigDecimal totalSales;
}

File: Faza6/FrontEnd/Angular/src/app/model/Reports/WeeklySalesReport.ts
import Big from "big.js";

export class WeeklySalesReport{
  username:string;
  weekNumber:Big;
  totalOrders:number;
  totalSales:Big;
  constructor(
    username:string,
    weekNumber:Big,
    totalOrders:number,
    totalSales:Big,
  ){
    this.username=username;
    this.weekNumber=weekNumber;
    this.totalOrders=totalOrders;
    this.totalSales=totalSales;
  }

}
